librocksdb_sys/src/lib.rs

```diff
@@ -734,6 +734,10 @@ extern "C" {
     pub fn crocksdb_options_set_delayed_write_rate(options: *mut Options, rate: u64);
     pub fn crocksdb_options_set_force_consistency_checks(options: *mut Options, v: bool);
     pub fn crocksdb_options_set_ratelimiter(options: *mut Options, limiter: *mut DBRateLimiter);
+    pub fn crocksdb_options_set_ratelimiter_bytes_per_second(
```

Comment:
Can we split this method into a `crocksdb_options_ratelimiter()` getter and a `crocksdb_ratelimiter_set_bytes_per_second()` function instead?